src/main/java/io/confluent/copycat/jdbc/DataConverter.java

```diff
@@ -118,30 +116,30 @@ private static void addFieldSchema(ResultSetMetaData metadata, int col,
       case Types.TINYINT:
```

Comment:
Can we match the type exactly for `TINYINT` and `SMALLINT`? Shouldn't those map to `int8` and `int16`?